prosessering-core/src/main/kotlin/no/nav/familie/prosessering/domene/Loggtype.kt

```diff
@@ -1,5 +1,5 @@
 package no.nav.familie.prosessering.domene
 
 enum class Loggtype {
-    FERDIG, FEILET, PLUKKET, BEHANDLER, KLAR_TIL_PLUKK, UBEHANDLET, AVVIKSHÅNDTERT, MANUELL_OPPFØLGING
+    FERDIG, FEILET, PLUKKET, BEHANDLER, KLAR_TIL_PLUKK, UBEHANDLET, AVVIKSHÅNDTERT, MANUELL_OPPFØLGING, KOMMENTAR
```

Comment:
Trenger man egen loggtype? Kunne MANUELL_OPPFØLGING vært brukt. Eller så synes jeg den burde hete PÅ_VENT eller noe mer beskrivende

Reply:
Eller jeg blander nok med forskjellige typer med status, og da vil nok dette fungere bra. Savner dog kanskje en egen status for de man venter på, eller sette de i manuell_oppfølging. Slik at det ikke blir så mye rødt. Jeg liker ikke rødt :D

Reply:
Hvis vi bruker `MANUELL_OPPFØLGING ` vil det bli vanskelig å skille tasker som setter dette automatisk til `MANUELL_OPPFØLGING ` ved feil i T`askworker.doFeilhåndtering`, men vi trenger kanskje ikke skille på det? Det er kanskje greit at det noen ganger ligger en stackTrace som kommentar.

Dette er jo LoggTypen og ikke status på tasken som fortsatt vil være feilet. Men kan godt endre hvis det er en mer beskrivende type for kommentarer.

Reply:
Jeg synes KOMMENTAR er bra som loggtypen. Jeg er mindre glad at de blir liggende igjen i FEILET og rekjørt hver morgen.

Reply:
nettopp, jeg ser behovet for begge varianter, noen ganger er det fint å kunne kommentere med at dette løser seg automatisk i morgen, da er det fint med automatisk rekjøring. og noen ganger er det greit å kommentere med at her må vi fikse "noe" først og da er det fint å kunne sette status til `MANUELL_OPPFØLGING` samtidig som vi kommenterer.

Reply:
Per dags dato mangler vi den funksjonaliteteten til å sette til manuell_oppfølging

Reply:
jeg la inn støtte for det nå